cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+    cy.get(':nth-child(1) > .form-control')
+      .type('luckezcy@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('testPassword1!');
```

Comment:
Hardcoding user credentials in the test files is not a good practice. It's better to store these values in a secure place like environment variables. You can then use `Cypress.env('username')` and `Cypress.env('password')` to access these values. This way, your test data remains secure and you can easily change it without modifying your test files.